firebase-functions/functions/src/choiceUpdated.ts

```diff
@@ -1,33 +1,49 @@
 ﻿import * as functions from 'firebase-functions';
 import * as admin from 'firebase-admin';
-import { getLosers, hasEveryoneChosen } from './choiceOperations';
+import { Player } from './models/player';
+import { getActiveLosers, hasEveryoneChosen } from './choiceOperations';
 
 export default functions.firestore.document('/players/{documentId}')
   .onUpdate(async (change: any, context: any) => {
     try {
       const roomId = await getCurrentRoomId(context.params.documentId as string);
-      const playersInSameRoom = (await admin.firestore().collection('players').where('room', '==', roomId).get()).docs;
+      const playersInSameRoom = await getActivePlayersInSameRoom(roomId);
```

Comment:
playersCurrentRoom I would say better than playersInSame but I won't push it :)

Reply:
Alright :)